charts/stable/owncast/questions.yaml

```diff
@@ -7,6 +7,14 @@ portals:
       - "$kubernetes-resource_configmap_portal_host"
     ports:
       - "$kubernetes-resource_configmap_portal_port"
+  admin_portal:
```

Comment:
Please don't use "admin_portal"

"Admin Portal" takes a lot of space and is weird.
Admin would be fine.

Reply:
It was copy-pasta from pihole, should we change it there also then?